c++/regenerate-bootstraps.sh

```diff
@@ -7,4 +7,5 @@ export PATH=$PWD/bin:$PWD:$PATH
 capnp compile -Isrc --no-standard-import --src-prefix=src -oc++:src \
     src/capnp/c++.capnp src/capnp/schema.capnp \
     src/capnp/compiler/lexer.capnp src/capnp/compiler/grammar.capnp \
-    src/capnp/rpc.capnp src/capnp/rpc-twoparty.capnp src/capnp/persistent.capnp
+    src/capnp/rpc.capnp src/capnp/rpc-twoparty.capnp src/capnp/persistent.capnp \
```

Comment:
Truly speaking I don't understand what I am doing here. I just tried to be consistent and I hoped for review feedback. I will try to address these comments the best I can next week.